.github/workflows/test_tools.yml

```diff
@@ -20,6 +20,10 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Install dependencies
-        run: pip install -r requirements.txt
+        # mypy version copied from .circleci/docker/common/install_conda.sh
+        run: |
+          set -eux
+          pip install -r requirements.txt
+          pip install mypy==0.770
```

Comment:
I think we discussed this before, but why did we decide against having mypy be a requirement?

Reply:
I don't remember; @pytorch/pytorch-dev-infra anyone know?